build/rust/README.md

```diff
@@ -0,0 +1,154 @@
+# Tucana - Protobuf gRPC Interface for Rust
+
+[![Crate](https://img.shields.io/crates/v/tucana.svg)](https://crates.io/crates/tucana)
+[![Documentation](https://docs.rs/tucana/badge.svg)](https://docs.rs/tucana)
+
+A Rust library providing Protobuf gRPC interfaces for internal service communication along with helper utilities for working with the generated types.
```

Comment:
I think this readme will show up on the crate docs, it should somewhere mention that it is the grpc interface for Code0. From that docs page, there isn't really a link back to the project for people that want to learn more about it.